libnmstate/state.py

```diff
@@ -51,7 +52,58 @@
 
 
 @total_ordering
-class RouteEntry(object):
+class StateEntry:
+    def __init__(self):
+        pass
+
+    def keys(self):
+        """
+        Return the tuple representing this entry, could be used for hashing or
+        comparing.
+        """
+        return ()
```

Comment:
This seems like an abstract, therefore it will fit to add here `raise NotImplementedError`.